app/features-json/build_json_controller.rb

```diff
@@ -191,7 +190,9 @@ def current_project
 
     # convert .log files that include the color information as ANSI code to plain text
     def convert_ansi_to_plain_text(data)
-      return data.gsub(/\e\[[0-9;]*m/, "")
+      data.scan(ANSI_PATTERN).inject("") do |str, match|
+        str << (match[3] || match[4])
```

Comment:
Could this potentially crash? If so, let's make sure to `rescue` the exception and fallback to plain text